app/src/main/java/com/github/h3lp3rs/h3lp/signin/SignInActivity.kt

```diff
@@ -95,12 +95,20 @@ class SignInActivity : AppCompatActivity() {
     companion object{
         @SuppressLint("StaticFieldLeak")
         lateinit var globalContext: Context
+        var userUid: String? = null
 
         /**
          * Getter on the global context
          */
         fun getGlobalCtx(): Context {
             return globalContext
         }
+
+        /**
+         * Getter on the userUid
+         */
+        fun getUid(): String? {
+            return userUid
+        }
```

Comment:
I am not sure if a getter is necessary here since userId is already public and no transformation actually occurs in the getter.

Reply:
You're right, I just copied the context getter 👍 
Should be done for a next refactoring